content/en/blog/2021/istio-4th-birthday/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Happy Birthday, Istio!"
+description: "Celebrating Istio’s 4th birthday."
+publishdate: 2021-05-24
+attribution: "Maria Cruz (Google), Aizhamal Nurmamat kyzy (Google)"
+keywords: [community,birthday,celebration]
+---
+
+## Celebrating Istio’s 4th birthday
+
+Four years ago today, the Istio project was born to the open source world. To celebrate this anniversary,
+we are hosting a week-long birthday celebration  that focuses on contributions to the Istio project that
+stem from using Istio in production. Read on to learn how to participate in this celebration and enter a
+chance to win some Istio swag.
+
+{{< image link="./1.png" caption="Istio's 4th Birthday!" >}}
+
+### A year of important developments for Istio
+
+Over the last 12 months, the Istio project has been very focused on the [day-0
+& day-1](https://dzone.com/articles/defining-day-2-operations) experience for
+users by actively listening to our users through UX surveys and GitHub issues.
+
+- We [simplified the control plane architecture](https://istio.io/latest/blog/2020/istiod/) and
+   made Istio easier to install, configure and upgrade.
+- We provided clarity and process to our feature status and promotion of features and APIs.
+- We simplified the debugging experience with various istioctl commands.
+- We expanded the mesh to services running in [VMs](https://istio.io/latest/news/releases/1.9.x/announcing-1.9/#virtual-machine-integration-beta)
```

Comment:
To satisfy the 7 lint link errors, all absolute links should be of the form [/news/releases/1.9.x/announcing-1.9/#virtual-machine-integration-beta]. See https://istio.io/latest/docs/releases/contribute/shortcodes/#add-links-to-other-pages.